### PR TITLE
Add initial macOS app

### PR DESCRIPTION
**Pull request by michael-loki**

### CODE REVIEW

**Comment by majestrate** [comment]
stray file? `ui-macos/.DS_Store`

**Comment by despair86** [comment]
ew resource forks

**Comment by despair86** [comment]
also, isn't the xcode/mpw project stuff machine specific, i.e. you can't take the project files into source control

File: ui-macos/lokinet/DNSManager.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        }
+    }
+
+    func setNewSettings() throws {
+        let netprocess = Process()
+        netprocess.executableURL = DNSManager.netSetup
+
+        netprocess.arguments = ["-setdnsservers", self.interface]
+        netprocess.arguments?.append("127.0.0.1")
+        netprocess.arguments?.append(contentsOf: oldDNSSettings)
```
Comment (majestrate) [inline ui-macos/lokinet/DNSManager.swift]: the dns server should be ONLY lokinet so that leaks don't make lookups return false positives.

**Comment by KeeJef** [comment]
Bump on this, should be included in either a point release or 0.6.0. What are we waiting on before merge?

**Comment by michael-loki** [comment]
@KeeJef I thought I pushed my windows fixes, done now.

**Pull request merged**